tests/components/unifi/conftest.py

```diff
@@ -52,6 +51,22 @@
     "uptime": 1562600160,
 }
 
+type ConfigEntryFactoryType = Callable[
+    [MockConfigEntry], Coroutine[Any, Any, MockConfigEntry]
```

Comment:
Why is the config entry a parameter for the factory? Sounds strange

Reply:
Copy paste mishap from deCONZ fixture. Good catch! Fixed